src/overlays/actors/ovl_En_Snowman/z_en_snowman.c

```diff
@@ -34,8 +34,8 @@ void EnSnowman_SetupDead(EnSnowman* this);
 void EnSnowman_Dead(EnSnowman* this, PlayState* play);
 void EnSnowman_SetupSplitDoNothing(EnSnowman* this);
 void EnSnowman_SplitDoNothing(EnSnowman* this, PlayState* play);
-void EnSnowman_SetupMarkForDeath(EnSnowman* this);
-void EnSnowman_MarkForDeath(EnSnowman* this, PlayState* play);
+void EnSnowman_SetupKill(EnSnowman* this);
+void EnSnowman_Kill(EnSnowman* this, PlayState* play);
```

Comment:
nice catch

Reply:
After I did a find-and-replace for `Actor_MarkForDeath`, I did a search for `MarkForDeath` by itself specifically to catch stuff like this. I expected to see more than this, though.

Reply:
They're in Bigslime/Minislime, and I don't feel comfortable enough with those actors to deal with that. I'll leave them for now.